### PR TITLE
Added Natural Disaster Prediction

### PR DESCRIPTION
**Pull request by infiniteoverflow**
Fixes issue #130 

I have used a Cyclical Learning Rates approach for training the dataset for 48 epochs resulting in an accuracy above 85%.
I have attached the plot in the comments below.

### CODE REVIEW

**Comment by infiniteoverflow** [comment]
ss1

**Comment by infiniteoverflow** [comment]
ss2

**Comment by Jayshah6699** [comment]
Adding here screenshots will not acceptable. You need to upload the ipynb file after running in your local machine or colab with generated output

**Comment by infiniteoverflow** [comment]
Yes , the output is visible in the Jupyter notebook. 
i added the screenshots here for just a reference.

**Comment by Jayshah6699** [comment]
You have not run the whole ipynb Do that first.

**Comment by infiniteoverflow** [comment]
Oh,i am sorry , i think the outputs got missed out. I will commit with the outputs soon !!

**Comment by infiniteoverflow** [comment]
@Jayshah6699 i have updated the notebook, kindly have a look. Thanks

**Comment by infiniteoverflow** [comment]
https://drive.google.com/drive/folders/12GSdRTX_iAPZCo_ziOqlqvuW7Ug2ag1T?usp=sharing

This is the link to the output generated by the code , it consists of the training plots and the model file. As this was around 200mb in size , i couldnt append this folder in the repo.

**Comment by Jayshah6699** [comment]
You can upload it on kaggle publically and add the link and just add ipynb file consisting output

**Comment by infiniteoverflow** [comment]
Thats the trained model file , should i upload that in kaggle ?

**Comment by Jayshah6699** [comment]
Yeah I was talking about that only

**Comment by infiniteoverflow** [comment]
As i am new to Kaggle , Can you please guide me through as in where i should upload the trained model file on kaggle ?

**Comment by infiniteoverflow** [comment]
@Jayshah6699 i have added the notebook to kaggle 
https://www.kaggle.com/aswin1871/natural-disaster-prediction

**Comment by Jayshah6699** [comment]
The notebook You have added on kaggle is just errors nothing else. Kindly upload the one which has no errors.

**Comment by infiniteoverflow** [comment]
Oops sorry , my bad. I have freshly ran the notebook which clearly shows the output :
https://www.kaggle.com/aswin1871/disaster-prediction

@Jayshah6699 sir , kindly have a look. Thank you

**Comment by Jayshah6699** [comment]
ping me on discord. We need to talk about this.
